docker-wine

```diff
@@ -609,6 +609,7 @@ else
 
         # Add Linux run args
         add_run_arg --env="DISPLAY"
+        add_run_arg --volume="$XDG_RUNTIME_DIR/$WAYLAND_DISPLAY:/tmp/$WAYLAND_DISPLAY"
```

Comment:
@macdabby maybe it's worth putting this behind a conditional so it's set (as per feedback from @scottyhardy)

```bash
	if [ -v WAYLAND_DISPLAY ]; then
		 add_run_arg --volume="$XDG_RUNTIME_DIR/$WAYLAND_DISPLAY:/tmp/$WAYLAND_DISPLAY"
	fi
```

Reply:
On my local system XDG_RUNTIME_DIR is /run, will `/tmp` always work ?